src/api.ts

```diff
@@ -24,4 +24,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | undefined) => get<User>(`/users/${userId}`);
```

Comment:
Why call `getUser` with `undefined`, just don't call it if we don't have the userId.

Reply:
This way, you are making a network request that is guaranteed to fail